migrations/applied_migrations

```diff
@@ -1,36 +1,35 @@
 #!/bin/bash
 
 echo ""
-echo "Applying migration ClaimPeriodStart"
+echo "Applying migration claimPeriod"
 
 echo "Adding routes to conf/app.routes"
 
 echo "" >> ../conf/app.routes
-echo "GET        /claimPeriodStart                  controllers.ClaimPeriodStartController.onPageLoad(mode: Mode = NormalMode)" >> ../conf/app.routes
-echo "POST       /claimPeriodStart                  controllers.ClaimPeriodStartController.onSubmit(mode: Mode = NormalMode)" >> ../conf/app.routes
+echo "GET        /claimPeriod                        controllers.claimPeriodController.onPageLoad(mode: Mode = NormalMode)" >> ../conf/app.routes
```

Comment:
we don't camel case endpoints.

`/claimPeriod` should be `/claim-period`

Reply:
you can ignore this file and actual routes have been updated accordingly